### PR TITLE
More terraria UI

### PR DESCRIPTION
**Pull request by heinermann**

### CODE REVIEW

**Comment by IntQuant** [comment]
Nice!
Larger scale might make it a bit too big in some cases maybe?

**Comment by heinermann** [comment]
I thought the point was to be annoyingly there.

**Comment by EvaisaDev** [comment]
i don't think we should make it bigger either personally

File: files/content/better_ui/better_ui.lua [hunk 1]
```diff
@@ -18,7 +18,7 @@ local pending_info = {}
 
 local module = {}
 
-local scale = 0.75
+local scale = 1
```
Comment (heinermann) [inline files/content/better_ui/better_ui.lua:21]:
```suggestion
local scale = 0.75
```

**Pull request merged**